ideascube/mediacenter/templatetags/mediacenter_tags.py

```diff
@@ -13,4 +13,6 @@ def preview_url(inst):
     elif inst.kind == inst.IMAGE:
         return inst.original.url
     else:
-        return static('mediacenter/document.svg')
+        if inst.kind == inst.OTHER:
+            return static('mediacenter/document.svg')
+        return static('mediacenter/default_{}.png'.format(inst.kind))
```

Comment:
Cool :)
I suggest generating a `default_other.png` instead of keeping a specific case for kind.OTHER, though :)